common/data_refinery_common/models/dataset_annotation.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.utils import timezone
+
+from data_refinery_common.models.managers import PublicObjectsManager
+
+
+class DatasetAnnotation(models.Model):
+    """ Semi-standard information associated with a Dataset """
+
+    class Meta:
+        db_table = "dataset_annotations"
+        base_manager_name = "public_objects"
+
+    # Managers
+    objects = models.Manager()
+    public_objects = PublicObjectsManager()
+
+    # Relations
+    dataset = models.ForeignKey("Dataset", blank=False, null=False, on_delete=models.CASCADE)
+
+    # Properties
+    data = JSONField(default=dict)
+
+    # Common Properties
+    is_public = models.BooleanField(default=True)
```

Comment:
Hmm, would it be an issue if people could query our API and see what data was downloaded by which IPs @jaclyn-taroni ? I think I remember overhearing discussions about people potentially being scared to use us in fear that what data they are downloading could be used to help scoop them? Is that actually a concern?

It's not actually an issue because this new model doesn't have a view so it's not accessible via the API. However, if it was a concern we could set public to False by default and leave a comment about how a view should not be created for this model in the future.

Reply:
But that `is_public` variable is only used internally right? Does it have anything to do with the objects being in the API? My interpretation of it was that if `is_public=False` is as if the object is removed from the db.

We also never expose the `email_address` field of the datasets, which is part of the model.

Reply:
My understanding is that is that `is_public` just controls whether or not the `DatasetAnnotation.public_objects` manager returns them or not. If you use `DatasetAnnotation.objects` it should return them whether they are public or not.

Reply:
> would it be an issue if people could query our API and see what data was downloaded by which IPs

Gut reaction here: I do not like the idea of that being a possibility.

Reply:
The base manager is the public objects manager so if it was false it shouldn't show up with `DatasetAnnotation.objects` I believe.